Release: npm@6.6.0

zkat opened the pull request:
[rendered changelog](https://github.com/npm/cli/blob/release-next/CHANGELOG.md)

zkat committed:
config: add figgyConfig for things that use figgy-pudding

zkat committed:
whoami: stop using npm-registry-client

zkat committed:
ping: stop using npm-registry-client

zkat committed:
deprecate: stop using npm-registry-client

zkat committed:
stars: stop using npm-registry-client

zkat committed:
star: stop using npm-registry-client

zkat committed:
view: stop using npm-registry-client

zkat committed:
audit: convert audit to new version of n-r-f

zkat committed:
npm-registry-fetch@3.8.0

zkat committed:
pacote@9.1.0

zkat committed:
outdated: stop using npm-registry-client

zkat committed:
dist-tag: stop using npm-registry-client

zkat committed:
anon-metrics: stop using npm-registry-client

zkat committed:
owner: stop using npm-registry-client

zkat committed:
team: stop using npm-registry-client

zkat committed:
utils: add otplease util for otp prompting

zkat committed:
access: stop using npm-registry-client

zkat committed:
hook: clean up and improve hook command

zkat committed:
org: add npm org command

Credit: @chrisdickinson
Credit: @zkat

zkat committed:
figgy-pudding@3.5.1

Credit: @zkat

zkat committed:
search: stop using npm-registry-client

zkat committed:
npm-profile@4.0.1

Credit: @zkat

zkat committed:
logout: stop using npm-registry-client

zkat committed:
login: update legacy auth to support latest npm-registry-fetch

zkat committed:
login: stop using npm-registry-client for SSO auth

zkat committed:
unpublish: stop using npm-registry-client

zkat committed:
semver@5.5.1

Credit: @isaacs

zkat committed:
publish: stop using npm-registry-client

zkat committed:
npm-registry-client: goodbye

zkat committed:
npm-registry-client@REMOVED

Ref: https://www.youtube.com/watch\?v\=kPIdRJlzERo
Credit: @zkat

zkat committed:
figgy-config: fix opts.color defaults because npm is weird

zkat committed:
libcipm@3.0.2

Credit: @zkat

zkat committed:
pacote: get rid of legacy pacoteOpts

zkat committed:
workers: fixup

zkat committed:
get-stream@4.1.0

larsgw committed:
install: fix checking for optional dep

Fixes: https://npm.community/t/2569

Credit: @larsgw
Reviewed-By: @iarna

zkat committed:
libnpm@1.4.0

zkat committed:
npm-profile@REMOVED

zkat committed:
libnpm@2.0.1

zkat committed:
config: remove some config checks

zkat committed:
install: fix circular dependency with config stuff

zkat committed:
test: update cache add test output

zkat committed:
test: remove get.js test for n-r-c

zkat committed:
config: fix passing of referer value

dmabupt committed:
config: support default user on IBM i (#65)

setuid() can not accept a 'nobody' parameter on IBM i.
The default user (QSECOFR) on IBM i has user id 0.

PR-URL: https://github.com/npm/cli/pull/65
Credit: @dmabupt
Reviewed-By: @zkat

GeorgeTaveras1231 committed:
docs: Fix typo in npm-token documentation (#71)

PR-URL: https://github.com/npm/cli/pull/71
Credit: @GeorgeTaveras1231
Reviewed-By: @zkat

larsgw committed:
pacote: pass git opts to pacote (#98)

Pass git binary path config to pacote.

Ref: zkat/pacote#164
Fixes: https://npm.community/t/3278
PR-URL: https://github.com/npm/cli/pull/98
Credit: @larsgw
Reviewed-By: @zkat

isaacs committed:
dist-tag: make 'ls' the default action (#106)

I keep typing `npm dist-tags` expecting it to print out a list of
dist-tags and instead it yells at me and that feels very un-npm-y.

PR-URL: https://github.com/npm/cli/pull/106
Credit: @isaacs
Reviewed-By: @zkat

BeniCheni committed:
utils: check npm.config existence in error-handler.js (#108)

Fixes: https://npm.community/t/npx-envinfo-preset-jest-fails-on-windows-with-a-stack-trace/2713
PR-URL: https://github.com/npm/cli/pull/108
Credit: @BeniCheni
Reviewed-By: @zkat

zkat committed:
licensee@5.0.0

zkat committed:
deps: move rimraf to prod deps

zkat committed:
doc: update changelog for npm@6.6.0

fharper reviewed (commented):
LGTM, just format the first bug link as the second one

zkat committed:
deps: alpha-order bundleDeps

zkat committed:
deps: fix up gitignore issues with devDeps

zkat committed:
deps: make prepare run `rebuild` to make sure bins from deps are there

Commit:
6.6.0-next.0

Commit:
update AUTHORS

zkat committed:
npm-audit-report@1.3.2

Ref: https://github.com/npm/npm-audit-report/pull/34
Credit: @melkikh

iarna committed:
scripts: Modernize maketest

Credit: @iarna

iarna committed:
test: maketest: Use promise based example common.npm call

Credit: @iarna

iarna committed:
scripts: maketest: Use newEnv for env production

Credit: @iarna

iarna committed:
test: Correct docs for fake-registry interface

Credit: @iarna

zkat committed:
profile: update profile to support new npm-profile API (#131)

PR-URL: https://github.com/npm/cli/pull/131
Credit: @zkat
Reviewed-By: @aeschright

larsgw committed:
deps: remove tink experiments (#129)

Remove tink experiments. As far as I know, these are not intended to be 
here.

Fixes: https://npm.community/t/4172
Credit: @larsgw
Reviewed-By: @zkat

adriangodong commented:
Is https://github.com/npm/cli/pull/113/commits/134207174652e1eb6d7b0f44fd9858a0b6a0cd6c missing from changelog?

zkat committed:
spdx-license-ids@3.0.3

Ref: https://github.com/npm/cli/pull/121

Commit:
aproba@2.0.0

Commit:
byte-size@5.0.1

Commit:
cacache@11.3.2

Commit:
chownr@1.1.1

Commit:
ci-info@2.0.0

Commit:
cli-table3@0.5.1

Commit:
is-cidr@3.0.0

Commit:
JSONStream@1.3.5

Commit:
libnpmhook@5.0.2

Commit:
marked@0.6.0

Commit:
npm-packlist@1.2.0

Commit:
unique-filename@1.1.1

Commit:
semver@5.6.0

Commit:
rimraf@2.6.3

Commit:
require-inject@1.4.4

Commit:
npm-pick-manifest@2.2.3

Commit:
query-string@6.2.0

Commit:
pacote@9.3.0

Commit:
readable-stream@3.1.1

Commit:
lru-cache@4.1.5

Commit:
deps: updating semver docs

johanneswuerbach committed:
Handle git branch references correctly (#123)

PR-URL: https://github.com/npm/cli/pull/123
Credit: @johanneswuerbach
Reviewed-By: @aeschright

zkat committed:
audit: report any errors above 400 as potentially not supporting audit (#128)

PR-URL: https://github.com/npm/cli/pull/128
Fixes: https://npm.community/t/npm-audit-fails-with-enoaudit-on-500-response/3629
Fixes: https://npm.community/t/npm-audit-error-messaging-update-for-401s/3983
Credit: @zkat 
Reviewed-By: @aeschright

anchnk committed:
cli,outdated: default homepage to an empty string (#124)

PR-URL: https://github.com/npm/cli/pull/124
Credit: @anchnk 
Reviewed-By: @aeschright

larsgw committed:
doc: fix npm-prefix description (#116)

PR-URL: https://github.com/npm/cli/pull/116
Fixes: https://npm.community/t/4054
Credit: @larsgw 
Reviewed-By: @aeschright

watilde committed:
chore: replace var with const/let in lib/repo.js (#119)

PR-URL: https://github.com/npm/cli/pull/119
Credit: @watilde 
Reviewed-By: @aeschright

Commit:
doc: update changelog for npm@6.6.0

Commit:
6.6.0-next.1

Commit:
update AUTHORS

Commit:
chore: Update package-lock.json for https tarball URLs

Commit:
doc: update changelog for npm@6.6.0

Commit:
6.6.0
